### ISSUE TITLE
Reinstall corrupted translator files automatically

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#903]

**Comment by zuphilip** [comment]
I am just curious: Was there any corrupted translator file recently distributed?

**Comment by dstillman** [comment]
No, it's not about distribution — we just sometimes see empty or otherwise corrupted files in people's data directories, from security software, OS bugs, disk corruption... One of our developers also had a few empty files in two different data directories on 5.0, so there may be an additional issue there that we haven't yet tracked down.
